src/sage/repl/ipython_extension.py

```diff
@@ -357,12 +373,23 @@ def cython(self, line, cell):
           - ``--use-cache``
           - ``--create-local-c-file``
           - ``--annotate``
+          - ``--view-annotate``
           - ``--sage-namespace``
           - ``--create-local-so-file``
           - ``--no-compile-message``, ``--no-use-cache``, etc.
 
           See :func:`~sage.misc.cython.cython` for details.
 
+          For ``--view-annotate``, the following additional choices are allowed:
+
+          - ``--view-annotate=none`` (default)
+          - ``--view-annotate=auto`` (same as ``--view-annotate``); select one of the
+            choices below automatically
+          - ``--view-annotate=webbrowser``; open the annotation in a web browser
+            (preferred if the Sage command line is used)
+          - ``--view-annotate=displayhtml``; display the annotation inline in the notebook
+            (preferred if the Sage notebook is used)
+
         - ``cell`` -- string; the Cython source code to process
```

Comment:
How about removing the lines 389 and 391 and instead use them to explain "automatically" for "auto" option?